loom/script/compiler/lsJitTypeCompiler.cpp

```diff
@@ -2389,7 +1803,7 @@ Expression *JitTypeCompiler::visit(FunctionLiteral *literal)
     inLocalFunction++;
 
     char funcname[256];
-    sprintf(funcname, "__ls_localfunction%i", functioncount++);
+    snprintf(funcname, 250, "__ls_localfunction%i", functioncount++);
```

Comment:
Correlates to buffer above, but shorter than having an arbitrary #define and simpler than a sizeof macro.